sdks/python/apache_beam/io/lyft/s3_and_kinesis.py

```diff
@@ -84,6 +85,16 @@ def get_windowing(self, inputs):
     def infer_output_type(self, unused_input_type):
         return bytes
 
+    def with_app_env(self, app_env):
```

Comment:
minor nit: given this is a public api, let's call this `with_app_environment` and avoid short names ?